src/material.cpp

```diff
@@ -843,7 +843,8 @@ void pyne::Material::from_text(std::string filename) {
     } else if (keystr == "APerM") {
       f >> valstr;
       atoms_per_molecule = pyne::to_dbl(valstr);
-    } else if (pyne::nucname::isnuclide(keystr)) {
+    } else if (pyne::nucname::isnuclide(keystr) ||
+               pyne::nucname::iselement(keystr)) {
```

Comment:
Correction. I've just read the relevant issue again and I agree this should fix it. Perhaps this is a good opportunity for  a quick test replicating #844 so that we don't reintroduce the bug?